class-wpgraphiql.php

```diff
@@ -0,0 +1,141 @@
+			);
+		}
+
+		/**
+		 * Renders WPGraphiQL admin page
+		 */
+		public function render_admin_page() {
+
+			if ( $this->is_wpgraphql_active() ) {
+				echo '<div class="wrap"><div id="wp-graphiql"></div></div>';
+			} else {
+				echo '<div class="wrap"><h1>This plugin requires WPGraphQL to be installed to work. Please install WPGraphQL (https://github.com/wp-graphql/wp-graphql) and visit this page again.</h1></div>';
+			}
+
+		}
+
+		/**
+		 * Gets the contents of the Create React App manifest file
+		 *
+		 * @return array|bool|string
+		 */
+		public function get_app_manifest() {
+			$manifest = file_get_contents( dirname( __FILE__ ) . '/assets/app/build/asset-manifest.json' );
+			$manifest = (array) json_decode( $manifest );
+			return $manifest;
+		}
+
+		/**
+		 * Gets the path to the stylesheet compiled by Create React App
+		 *
+		 * @return string
+		 */
+		public function get_app_stylesheet() {
+			$manifest = $this->get_app_manifest();
+			if ( empty( $manifest['main.css'] ) ) {
+				return '';
+			}
+			return WPGRAPHIQL_PLUGIN_DIR . 'assets/app/build/' . $manifest['main.css'];
+		}
+
+		/**
+		 * Gets the path to the built javascript file compiled by Create React App
+		 *
+		 * @return string
+		 */
+		public function get_app_script() {
+			$manifest = $this->get_app_manifest();
+			if ( empty( $manifest['main.js'] ) ) {
+				return '';
+			}
+			return WPGRAPHIQL_PLUGIN_DIR . 'assets/app/build/' . $manifest['main.js'];
+		}
+
+		/**
+		 * Returns path to asset manifest json
+		 */
+		public function get_app_script_helpers() {
+			$manifest = $this->get_app_manifest();
+			if ( empty( $manifest['main.js'] ) ) {
+				return '';
+			}
+			return WPGRAPHIQL_PLUGIN_DIR . 'assets/js/wp-graphiql-helpers.js';
+		}
+
+		/**
+		 * Enqueues the stylesheet and js for the WPGraphiQL app
+		 */
+		public function enqueue_react_app() {
+
+			/**
+			 * Only enqueue the assets on the proper admin page, and only if WPGraphQL is also active
+			 */
+			if ( strpos( get_current_screen()->id, 'wp-graphiql/wp-graphiql' ) && $this->is_wpgraphql_active() ) {
+
+				wp_enqueue_style( 'wp-graphiql-theme', '//cdnjs.cloudflare.com/ajax/libs/codemirror/5.23.0/theme/solarized.css', array(), false, false );
```

Comment:
@kidunot89 looks like this PR is doing a lot more than adding a refresh button 😂 

I'll pull this down and check it out

Reply:
Yea, I forgot to take out the theme I set, my bad.